N/NCTiles/Compat.toml

```diff
@@ -0,0 +1,2 @@
+[0]
+julia = "0.7.0-*"
```

Comment:
Use
```toml
[compat]
julia = "0.7, 1"
```
instead

Reply:
Should be all set now

Reply:
It doesn't seem to have changed here. Do you need to retrigger?

Reply:
Just tried `@JuliaRegistrator register()` again but get this error message:

`Error while trying to register: "Tag with name 0.1.4 already exists and points to a different commit"`

and if I increase the version # in `Project.toml` then a new PR is created.

Apologies if I am missing something obvious, but could you please advise on how I should proceed

Reply:
You created a tag on the repo before this package was registered. The registation bot won't let you register a version that was already tagged if the thing you're trying to register doesn't match what's already tagged. The solution is to delete the incorrect tag/release on your project and then retrigger the registration bot.

Reply:
Thanks. Did not know I could delete a tag